### ISSUE TITLE
The pre-trained models are not available at the link

### ISSUE THREAD

**Issue by rondi201** [thswodnjs3/CSTA#4]
The pre-trained models are not available at the link from README. When clicking on the link, a 404 error appears at Google drive. Is it still possible to get the weights of the pre-trained models?

**Comment by thswodnjs3** [comment]
I'm very sorry, I found out that the datasets and pre-trained weights are no longer available.
I guess I accidentally deleted them.
You can download datasets from other repositories (e.g., PGL-SUM, A2Summ).
However, you need to obtain the pre-trained weights yourself by using the seed number specified in the README section.(https://github.com/thswodnjs3/CSTA?tab=readme-ov-file#pre-trained-models)
I'm sorry for the inconvenience.
If you fail to see a similar performance with that seed number, please contact me again.

**Comment by miniayu** [comment]
Thank you for your work, author. I would like to ask why the results I got from retraining are slightly different, namely [FINAL - SumMe]Kendall:0.227, Spear:0.253 and [FINAL - TVSum]Kendall:0.189, Spear:0.248. I look forward to your reply.

**Comment by thswodnjs3** [comment]
I apologize for my late reply.
Are your results substantially different from the ones reported in the paper?
What values did you observe?
